### PR TITLE
Gauntlet Recommendations to Lower stMATIC/MaticX non-emode LT/LTV on Polygon v3

### PR DESCRIPTION
**Pull request by reem**

### CODE REVIEW

**Review by ringwraith10 — APPROVED** [review]
lgtm

File: src/AaveV3Update_20231002matic/AaveV3Update_20231002matic.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| Polygon | stMATIC | Lower non-emode LTV| 50%| 45%
+| Polygon | MaticX | Lower non-emode LT| 67%| 62%
+| Polygon | MaticX | Lower non-emode LTV| 58%| 45%
+
+
+
+## Implementation
+
+The proposal implements changes on Polygon v3 using the following pre-deployed payloads:
+  - [Polygon](https://github.com/GauntletNetworks/aave-proposals/blob/628e17e84c9d6b8177adaad22aa87a69a045e829/src/AaveV3Update_20231002matic/AaveV3Polygon_20231002matic.sol)
```
Comment (sakulstra) [inline src/AaveV3Update_20231002matic/AaveV3Update_20231002matic.md:33]: if you want our ci to pin this, i think you need to use bgd-labs/aave-proposals/blob/main, but i guess you already pinned manually, so doesn't matter

File: src/AaveV3Update_20231002matic/AaveV3Polygon_20231002matic.sol [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import {IEngine,EngineFlags,Rates} from 'aave-helpers/v3-config-engine/AaveV3PayloadBase.sol';
+import {
+  AaveV3PayloadPolygon,
+  AaveV3PolygonAssets
+} from 'aave-helpers/v3-config-engine/AaveV3PayloadPolygon.sol';
+
+/**
+  * @title Gauntlet Recommendations to Lower stMATIC/MaticX non-emode LT/LTV on Polygon v3
+  * @author Gauntlet
+  * - Discussion: https://governance.aave.com/t/arfc-gauntlet-recommendation-to-lower-stmatic-maticx-non-emode-lt/14859
```
Comment (sakulstra) [inline src/AaveV3Update_20231002matic/AaveV3Polygon_20231002matic.sol:13]: we usually recommend to also link the snapshot, if it exists: https://snapshot.org/#/aave.eth/proposal/0x151df3127d187c203ff3a975d8ab9e34493d77edf67ec6884dc086d5aea9ce89

File: src/AaveV3Update_20231002matic/AaveV3Update_20231002matic.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
```
Comment (sakulstra) [inline src/AaveV3Update_20231002matic/AaveV3Update_20231002matic.md:34]: we usually recommend putting a references section with links to discussion and snapshot as not all governance uis interpret the discussions yaml style meta at the top

**Review by sakulstra — COMMENTED** [review]
overall looks fine, but just checking the snapshot it actually failed:
https://snapshot.org/#/aave.eth/proposal/0x151df3127d187c203ff3a975d8ab9e34493d77edf67ec6884dc086d5aea9ce89
No option passed 320k votes, which is defined as the quorum as i read the rules: https://governance.aave.com/t/arfc-arfc-and-temp-check-framework/13828

**Comment by tuanyuan2008** [comment]
> overall looks fine, but just checking the snapshot it actually failed:
> https://snapshot.org/#/aave.eth/proposal/0x151df3127d187c203ff3a975d8ab9e34493d77edf67ec6884dc086d5aea9ce89
> No option passed 320k votes, which is defined as the quorum as i read the rules: https://governance.aave.com/t/arfc-arfc-and-temp-check-framework/13828

Hi @sakulstra, we are moving forward with option 1 for lowering stMatic/MaticX LT/LTV because option 2 (225K votes) is more aggressive than option 1 (248K votes), so those who voted for option 2 would also be aligned with option 1. Given option 1 + option 2 (473K votes combined) is past quorum, we think it makes sense to move forward with Option 1.

**Review by sakulstra — APPROVED** [review]

**Review by rustboyar — DISMISSED** [review]

**Review by sakulstra — APPROVED** [review]

**Review by rustboyar — APPROVED** [review]

**Pull request merged**